src/markeritem.h

```diff
@@ -4,6 +4,7 @@
 #include <QGraphicsRectItem>
 #include <QObject>
 #include <QColor>
+#include <QPalette>
```

Comment:
Don't think you need this?

Reply:
You are probably right. I WAS using it but I think I forgot to delete this.